src/_includes/head.hbs

```diff
@@ -0,0 +1,32 @@
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <title>{{#if htmlWebpackPlugin.options.site.title}}{{htmlWebpackPlugin.options.site.title}}{{else}}{{htmlWebpackPlugin.options.site.default_title}}{{/if}}</title>
+  <meta name="description" content="{{#if htmlWebpackPlugin.options.site.description}}{{htmlWebpackPlugin.options.site.description}}{{else}}{{htmlWebpackPlugin.options.site.default_description}}{{/if}}">
+
+  {{!-- <link rel="canonical" href="{{ page.url | replace:'index.html', '' | prepend: site.url }}"> --}}
+  {{!-- <link rel="icon" href="{{ 'img/favicon.ico' | prepend: site.baseurl }}"> --}}
```

Comment:
Aree we bringing these back once you update how the URLs work?

Reply:
Yes, left the commented out while we work on it.